src/main/scala/me/shadaj/romeojuliet/Tragedy.scala

```diff
@@ -21,6 +21,53 @@ case class YouHaveToMarry(that: ActorRef)
 case object HelpMe
 
 case object SleepingPotion
+case class WakingUp(actorRef: ActorRef)
+
+object Tragedy {
```

Comment:
`Props` should not be created from inside an actor instance – that's important for remoting. Instead it's a pattern to have `Props`-factories in the companion object.